### ISSUE TITLE
Inaccurate explanation of spec for touch-action css property

### ISSUE THREAD

**Issue by patrickhlauke** [jquery/PEP#101]
> According to the spec, the touch-action css property determines if an element will receive PointerEvents.

Unless I'm misreading the spec and my own tests, this explanation is not quite accurate. Pointer Events are always fired on elements - touch-action only defines (per spec) what happens to default behavior, i.e. if the event is consumed entirely (similar to preventDefault-ing a touchstart/touchmove/touchend) or if it will fire and then carry on with its normal default behavior.

> Specifically, only elements with the styling that have the touch-action value of none will receive events.

Following on from the above, this is also not quite accurate. values of pan-x and pan-y filter out which movement can be consumed (i.e. the event fires, but still triggers default browser behavior for x or y movement), so it's not just value of none that is relevant.

In short, I think the sentence in the explanation should be changed to reflect the fact that, rather than being based on the spec, the limitations are design choices done by this polyfill itself (as is hinted at later on with "This is to maintain compositiong scrolling optimizations where possible"). It's a valid reason to limit the polyfill, but it's confusing that it's implied this is in line with the spec.
